.github/workflows/ci.yml

```diff
@@ -35,10 +35,10 @@ jobs:
         uses: actions/setup-node@v3.6.0
 
       # See https://github.com/actions/setup-java
-      - name: Setup Java 8
+      - name: Setup Java 11
         uses: actions/setup-java@v3.11.0
         with:
-          java-version: '8'
+          java-version: '11'
           distribution: 'adopt'
```

Comment:
Can we change this to 'corretto' at the same time? Not sure why adopt was originally used, but feels like a good chance to align with our best practice here.